src/main/resources/entity-types/finance/drv_ledger_details.json5

```diff
@@ -0,0 +1,340 @@
+      visibleByDefault: false,
+      valueGetter: "ldger.jsonb ->> 'ledgerStatus'",
+      filterValueGetter: '"left"(lower(${tenant_id}_mod_finance_storage.f_unaccent(ldger.jsonb ->> \'ledgerStatus\'::text)), 600)',
+      valueFunction: '"left"(lower(${tenant_id}_mod_finance_storage.f_unaccent(:value)), 600)'
+    },
+    {
+      name: 'currency',
+      dataType: {
+        dataType: 'stringType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "ldger.jsonb ->> 'currency'"
+    },
+    {
+      name: 'acq_unit_ids',
+      dataType: {
+        dataType: 'arrayType',
+        itemDataType: {
+          dataType: 'rangedUUIDType',
+        },
+      },
+      queryable: false,
+      visibleByDefault: false,
+      valueGetter: "ldger.jsonb ->> 'acqUnitIds'",
+      filterValueGetter: "lower(${tenant_id}_mod_finance_storage.f_unaccent(ldger.jsonb ->> 'acqUnitIds'::text))",
+      valueFunction: 'lower(${tenant_id}_mod_finance_storage.f_unaccent(:value))'
+    },
+    {
+      name: 'restrict_encumbrance',
+      dataType: {
+        dataType: 'booleanType',
+      },
+      values: [
+        {
+          label: 'True',
+          value: 'true',
+        },
+        {
+          label: 'False',
+          value: 'false',
+        },
+      ],
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "ldger.jsonb ->> 'restrictEncumbrance'"
+    },
+    {
+      name: 'restrict_expenditures',
+      dataType: {
+        dataType: 'booleanType',
+      },
+      values: [
+        {
+          label: 'True',
+          value: 'true',
+        },
+        {
+          label: 'False',
+          value: 'false',
+        },
+      ],
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "ldger.jsonb ->> 'restrictExpenditures'"
+    },
+    {
+      name: 'created_date',
+      dataType: {
+        dataType: 'dateType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "ldger.jsonb -> 'metadata' ->> 'createdDate'"
+    },
+    {
+      name: 'updated_date',
+      dataType: {
+        dataType: 'dateType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "ldger.jsonb -> 'metadata' ->> 'updatedDate'"
+    },
+    {
+      name: 'created_by_user_id',
+      dataType: {
+        dataType: 'rangedUUIDType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "ldger.jsonb -> 'metadata' ->> 'createdByUserId'"
+    },
+
+    {
+      name: 'updated_by_user_id',
+      dataType: {
+        dataType: 'rangedUUIDType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "ldger.jsonb -> 'metadata' ->> 'updatedByUserId'"
+    },
+    {
+      name: 'allocated',
+      dataType: {
+        dataType: 'numberType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "ldger.jsonb ->> 'allocated'"
```

Comment:
This field is a numberType so we need to return it as a float instead of as text. Something like "(ldger.jsonb -> 'allocated')::float" should work.